shopinvader/models/sale.py

```diff
@@ -45,6 +45,7 @@ class SaleOrder(models.Model):
         compute="_compute_shopinvader_state",
         store=True,
     )
+    shopinvader_to_be_recomputed = fields.Boolean()
```

Comment:
do we really need a field?
It seems that is used only in the cart service: can't we rely on a ctx key?

Reply:
This is a try. I explore possibilities to get quick response from add_item service.

IMHO using context when you try to improve performance is a bad idea.

Reply:
> IMHO using context when you try to improve performance is a bad idea.

why? you need a flag to control a behavior. There's nothing really special.
It doesn't really matter where the flag comes from as long its state is reliable.
If you say "there are other cases where this won't work like 1,2,3" that's another story.

Reply:
It's better to use parameters as changing context is creating new environment (< v13)